src/screens/FeedScreen/Footer.js

```diff
@@ -103,12 +103,12 @@ const styles = StyleSheet.create({
 
 const handleTextCountColor = (totalVote) => {
   if (totalVote > 0) {
-    return COLORS.holyTosca;
+    return '#00ADB5';
   }
   if (totalVote < 0) {
-    return COLORS.red;
+    return '#FF2E63';
   }
-  return COLORS.gray1;
+  return '#C4C4C4';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return values in the `handleTextCountColor` function have been replaced with hardcoded color codes. It's generally a good practice to use named constants for these values to improve readability and maintainability.
```diff
-    return COLORS.holyTosca;
+    return '#00ADB5';
-    return COLORS.red;
+    return '#FF2E63';
-  return COLORS.gray1;
+  return '#C4C4C4';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->